forum/qa-theme/SnowFlat/js/selectLanguage.js

```diff
@@ -0,0 +1,66 @@
+document.addEventListener("DOMContentLoaded", () => {
+    if (typeof CKEDITOR != 'undefined') {
+        CKEDITOR.on( 'instanceReady', function() {
+            const categories = [
+                {category : 'HTML i CSS', language: 'xml'},
+                {category : 'C i C++', language: 'cpp'},
+                {category : 'JavaScript, jQuery, AJAX', language: 'jscript'},
+                {category : 'PHP, Symfony, Zend', language: 'php'},
+                {category : 'SQL, bazy danych', language: 'sql'},
+                {category : 'C# i .NET', language: 'csharp'},
+                {category : 'Java', language: 'java'},
+                {category : 'Python, Django', language: 'python'},
+            ]
+            const codeBlock = document.getElementById( 'cke_43' );
+            let clickHandler;
+            if ( location.href.includes( '/ask' ) ) {
+                clickHandler = () => {
+                    const firstCategory = document.querySelector( '#category_1' );
+                    const selectedFirstOption = firstCategory.children[ firstCategory.selectedIndex ];                       	
+                    if(selectedFirstOption.textContent === 'Programowanie') {
+                        const firstCategory = document.querySelector( '#category_2' );

```

Comment:
Sorry, że się czepiam, ale czy tutaj nie powinna być nazwa secondCategory? Numer w id to sugeruje, a firstCategory masz w linijce 18.

Reply:
Oh, rzeczywiście. Poprawione.